test/docfx.Test/common/TestFramework.cs

```diff
@@ -32,6 +32,8 @@ protected override ITestFrameworkExecutor CreateExecutor(AssemblyName assemblyNa
             Log.ForceVerbose = true;
 
             MakeDebugAssertThrowException();
+            if (Directory.Exists(AppData.MutexRoot))
```

Comment:
Is this intentional?

Reply:
yes, to cleanup mutex folder for my unit tests

Reply:
will remove this line